index.js

```diff
@@ -146,6 +146,7 @@ class TestingFramework {
     }
 
     _send(spec, sender, message) {
+        console.log(`SEND: ${message.text}`);
```

Comment:
Not all messages have `text`: https://github.com/Icenium/nchat-web-chat/blob/master/client/src/chat/Chat.tsx

Add logging for file and location

Reply:
Changed logging to include full message(json)